plugins/woocommerce/src/Admin/Features/Settings.php

```diff
@@ -89,6 +89,15 @@ public function enqueue_settings_editor_styles() {
 			);
 			// phpcs:enable WordPress.WP.EnqueuedResourceParameters.MissingVersion
 			wp_enqueue_style( 'wp-gutenberg-posts-dashboard' );
+
+			// phpcs:disable WordPress.WP.EnqueuedResourceParameters.MissingVersion
+			wp_register_style(
+				'wp-gutenberg-edit-site',
+				gutenberg_url( 'build/edit-site/style.css', __FILE__ ),
```

Comment:
Not sure if we should do the same here, just wanted to mention that CYS has recently moved away from using .edit-site class names to avoid conflicts and possible breakages as Gutenberg components in https://github.com/woocommerce/woocommerce/pull/52426.

Reply:
Interesting. Thanks. Including that file meant I didn't need to apply much CSS at all. Can we keep an eye on this?

Reply:
Yeah, I'll keep an eye on it.